databricks_cli/sdk/v1_service.py

```diff
@@ -0,0 +1,87 @@
+# Databricks CLI
+# Copyright 2021 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import copy
+
+from databricks_cli.sdk.version import OLD_API_VERSION
+
+
+class APIV1Client:
```

Comment:
Should we use "new-style" classes here? https://stackoverflow.com/questions/4015417/why-do-python-classes-inherit-object

Reply:
By new style do you mean using `(object)`? It seems that this matters if we want to maintain compatibility with python 2. So in Python3 `object` is implicitly inherited by default.